src/apis/printInBackground.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Print the current document.
+ * @method WebViewerInstance#printInBackground
+ * @param {boolean} [allPages=false] If true, all pages will be printed
+ * @param {Array.<number>} [pagesToPrint=[]] If allPages is false and pagesToPrint is passed in, will print only the pages passed in
+ * @param {boolean} [includeAnnotations=false] If true, will print the documents with the annotations
+ * @param {boolean} [includeComments=false] If true, will append comments to the document printed
```

Comment:
All the 4 options are optional but if I just call `instance.printInBackground()` I will see
```
webviewer-ui.min.js:50 No pages were passed in to be printed. Please pass either allPages set to true or a range of pages in pagesToPrint.
```
in the console. I think this essentially means that `allPages: true` or `pagesToPrint: ...` is a required option to use this API, which contradicts the documentation.

I think we should either change the documentation here or change the defaultOptions in the code.